### PR TITLE
Inicial Project Components

### PR DESCRIPTION
**Pull request by Nicolaswg**
# Main Point 🥇 

🦅 
> In this new PR we created a React project with `npx create-react-app`. This is the first step 👞 for building a completed React & Redux app.

## Project Schedule 📄 
- Add basic React project Structure.
- Organize Application structure with Redux [duck pattern](https://github.com/erikras/ducks-modular-redux).
- Add[ React Router ](https://reactrouter.com/web/guides/quick-start)Functionality [ Router, Routes, Route, Link ]

### Section of the Project ✏️ 
<h2>Book Section</h2>

- Created book section Components
- Add Navigation React Router functionality
- Add `Hard-code` book data as an example

<h2>Categories Section</h2>

- Created simple component
- Add `In construction text`

## Set up 💻 
- Clone the PR repo Branch https://github.com/Nicolaswg/react-bookstore.git
- Open the cloned repo command line and type `npm install`
- Run `npm start` and check your browser at port(3000)

<h2>Eat - Sleep - Code - Repeat</h2>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (omar-labana) [inline README.md]: - [x] Kindly, update your README to better represent the project. Please use the MV template.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Nicolaswg, 

I like the energy put into this pr! 👌
You have implemented all the required features without any bugs or errors. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Nicolaswg) [reply README.md]: Thank a lot for your review 😄

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Nicolaswg, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**